### ISSUE TITLE
Citra core crashes when exited ingame

### ISSUE THREAD

**Issue by hyarsan** [libretro/citra#11]
When you exit in the Citra core ( Via some game or start button in some homebrew for example ), it crashes RetroArch instead of it returning back to the RetroArch menu! It should use RETRO_ENVIRONMENT_SHUTDOWN instead **I think**.
`
#define RETRO_ENVIRONMENT_SHUTDOWN      7  /* N/A (NULL) --
                                            * Requests the frontend to shutdown.
                                            * Should only be used if game has a specific
                                            * way to shutdown the game from a menu item or similar.
                                            */
`

That sounds alot like what is needed.

**Comment by j-selby** [comment]
The crashing is unrelated to any LibRetro callback - there is a issue somewhere else in either Citra or RetroArch (which is unlikely). Closing as duplicate of #6

**Comment by hyarsan** [comment]
I didn't mean closing through "Close Content" unlike #6 
What I mean is when homebrew says you can quit by pressing the start button or by tapping on some in game button, basically what RETRO_ENVIRONMENT_SHUTDOWN is used for.

**Comment by j-selby** [comment]
Ah, right. I understand. This is unimplemented functionality in Citra, so this won't happen in a while I don't think.

**Comment by j-selby** [comment]
To follow up on this, did this occur with a specific game?

**Comment by hyarsan** [comment]
I don't think so. I have tried to exit 3 homebrew so far and it keeps crashing when I try to exit it like I would on an real console ( With some ingame button that says quit or commonly the start button next to the select )
With 3 homebrew I mean: Simple-Calc, BlargSNES & FidgetSpinner3DS because I have no life.

**Comment by j-selby** [comment]
Status update: blocked by https://github.com/citra-emu/citra/issues/1663
